test/index.js

```diff
@@ -2,12 +2,6 @@ var OSRM = require('../');
 var test = require('tape');
 var berlin_path = require('./osrm-data-path').data_path;
 
-test('constructor: throws if new keyword is not used', function(assert) {
```

Comment:
Since there is no reason not to support non-construct calls. So Instead of throwing I added this, take a look:
https://github.com/Project-OSRM/node-osrm/blob/ea36088732c7b3100ffea1192ca3978c10c767bd/src/node_osrm.cpp#L54-L76